releasenotes/notes/0.25/discrete-pulse-library-deprecation-3a95eba7e29d8d49.yaml

```diff
@@ -1,14 +1,16 @@
 ---
-features:
+features_pulse:
   - |
-    The :class:`~qiskit.pulse.SymbolicPulse` library was extended. The new pulses in the library are:
+
+    The :class:`~qiskit.pulse.SymbolicPulse` library was extended. The new pulse functions
+    in the library are::
 
       * :func:`~qiskit.pulse.library.GaussianDeriv`
       * :func:`~qiskit.pulse.library.Sech`
       * :func:`~qiskit.pulse.library.SechDeriv`
       * :func:`~qiskit.pulse.library.Square`
 
-    The new functions return a :class:`ScalableSymbolicPulse`, and match the functionality
+    The new functions return a :class:`~qiskit.pulse.ScalableSymbolicPulse` instance, and match the functionality
     of the corresponding functions in the discrete pulse library, with the exception of
-    `Square()` for which a phase of :math:`2\\pi` shifts by a full cycle (contrary to the
-    discrete `square()` where such a shift was induced by a :math:`\\pi` phase).
+    ``Square()`` for which a phase of :math:`2\\pi` shifts by a full cycle (contrary to the
```

Comment:
Is this 
```
:func:`.symbolic_pulses.Square`
```
and
```
:func:`.discrete.square`
```
? 🙂

Reply:
No idea 😄

Reply:
I am not sure about the sphinx scoping, but yes `~qiskit.pulse.Square` and `~qiskit.pulse.library.discrete.square`.